cmd/common.go

```diff
@@ -614,3 +614,30 @@ func exportCodesignFiles(tool Tool, archivePath, outputDirPath string) error {
 
 	return nil
 }
+
+func trimProjectpath(projpth string) string {
+	if len(projpth) == 0 {
+		return projpth
+	}
+	characterSetToRemove := []string{"\n", `'`, `"`}
+	for {
+		projpth = strings.Trim(projpth, "\n")
+		projpth = strings.Trim(projpth, `'`)
+		projpth = strings.Trim(projpth, `"`)
+		projpth = strings.TrimSpace(projpth)
+
+		if !contains(characterSetToRemove, projpth[:1]) && !contains(characterSetToRemove, projpth[len(projpth)-1:]) {
```

Comment:
additionally, we don't want to remove every occurance of `'` and `"` I believe

the project can include `'` or `"` in the file name, as far as I know

we only want to remove the outmost quotes

so, a single `strings.Trim(strings.Trim(...))` would be enough instead of the `for`, IMO

Reply:
beside of that, if you ever will need to use something like your contains function, we have that in a package already:
`sliceutil.IsStringInSlice(slice, string)`